src/options.jl

```diff
@@ -245,6 +246,7 @@ function validate_solver_options_and_add_default_values!(options::Options)::Opti
     check_aliases_and_add_default_value!(dict, dict_copy, [:max_jumps], 5)
     check_aliases_and_add_default_value!(dict, dict_copy, [:fixpoint_check], true)
     check_aliases_and_add_default_value!(dict, dict_copy, [:n], nothing)
+    check_aliases_and_add_default_value!(dict, dict_copy, [:init], Tuple{Int64,LazySets.LazySet{Float64}}[])
```

Comment:
`Int64` -> `Int` (`Int64` falls back to `Int64` on 64-bit systems, but `Int` is shorter and cleaner)